composer.json

```diff
@@ -26,6 +26,7 @@
     "require-dev": {
         "symfony/yaml": "~2.3|~3.0|~4.0",
         "symfony/console": "~2.3|~3.0|~4.0",
+        "symfony/phpunit-bridge": "^3.3",
         "phpunit/phpunit": "^4.8"
```

Comment:
We could drop this dependency then, right?

Reply:
indeed